fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PipeAnalyzer.java

```diff
@@ -66,12 +66,12 @@ public class PipeAnalyzer {
                     .add(PropertyAnalyzer.PROPERTIES_WAREHOUSE)
                     .build();
 
-    public static void analyzePipeName(PipeName pipeName, ConnectContext context) {
+    public static void analyzePipeName(PipeName pipeName, String db) {
```

Comment:
It's recommended to use `insertedDbName` or `expectedDbName` instead of `db`, I think it's much clearer.

Reply:
How about a common name `defaultDbName`?

Reply:
I think default is not good, because we of course can set a different value rather than a default value.